task_02/index.html

```diff
@@ -1,11 +1,180 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
 </head>
+
 <body>
-    Task 2
+  <div class="wrapper">
+    <!-- Desktop navigation -->
+    <nav role="navigation" class="desktop-navigation">
+      <div class="nav-wrapper">
+        <div class="list-item"><a href="#" class="links">Home</a></div>
+        <div class="list-item"><a href="#" class="links">Approach</a></div>
+        <div class="list-item"><a href="#" class="links">Work</a></div>
+        <div class="list-item"><a href="#" class="links">People</a></div>
+        <div class="list-item">
+          <a href="#" class="links" id="disabled">Jobs</a>
+        </div>
+        <div class="list-item">
+          <a href="#" class="links" id="disabled">Blog</a>
+        </div>
+        <div class="list-item">
+          <a href="#" class="links" id="disabled">Contact</a>
+        </div>
+      </div>
+    </nav>
+
+    <!-- Mobile nav -->
+    <nav role="navigation" class="mobile-navigation">
+      <div id="menuToggle">
+        <input type="checkbox" id="toggleCheckbox" onclick="moveText()">
```

Comment:
It's placed on top of span element, which represents hamburger menu, and tracks whether or not it's checked. If checked, it moves content section and opens the menu. It's z-index is bigger than span's and since it's opacity is 0 it seems like we're clicking on that span element.